src/reference/90-Developers-Guide/06-Datatype/00.md

```diff
@@ -5,12 +5,12 @@ out: Datatype.html
 Datatype
 --------
 
-sbt-datatype is a code generation library and an sbt autoplugin that generates
+`sbt-datatype` is a code generation library and an sbt autoplugin that generates
```

Comment:
Why sbt-datatype formatted this way? Shouldn't the monospace be reserved for code snippets and literals?